src/com/xilinx/rapidwright/util/BlackboxPopulator.java

```diff
@@ -132,11 +145,11 @@ private static boolean clearTargetSiteInsts(Module mod, Site proposedAnchorSite,
 
     /**
      * Determine if the given net is a clock net.
-     *
+     * Net.isClockNet() need the source of a net. This method checks the sink instead.
      * @param net The net to check
      * @return True if the net is a clock net
      */
-    public static boolean isClockNet(Net net) {
+    private static boolean driveClockPin(Net net) {
```

Comment:
Perhaps this should be `hasClockPinSink(Net net)`?

Reply:
changed